tests/integration/helpers.py

```diff
@@ -161,10 +161,6 @@ async def generate_anycharm_relation(
     any_charm_src_overwrite = {
         "any_charm.py": any_charm_content,
         "dns_record.py": dns_record_content,
-        # It's okay to write to /tmp for these tests, so # nosec is used
```

Comment:
Why this change?

Reply:
I was trying to understand why the file was overwritten every time I tried to edit it when the charm was already deployed. The location has nothing to do with it but since `/srv` doesn't trigger the static analysis warnings, I kept it